cmd/px-node-wiper/px-node-wiper.sh

```diff
@@ -104,6 +104,18 @@ else
   echo "-removedata argument not specified. Not wiping the drives"
 fi
 
+# Last check to see there are left over oci mounts
+MOUNTS=$(nsenter --mount=$HOSTPROC1_NS/mnt -- mount)
```

Comment:
what would happen if the nsenter command itself fails? would the script exit with no zero exit code?

I see if we are checking for the exit code on line 109. My worry is: Is it possible for line 108 itself to cause the script to exit?

Reply:
Command failure in this script does not exit the px-node-wiper.sh.  It may display an error but either way the exit code with be != 0 and we will skip the mount check.